docs/debian/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM debian:jessie
+MAINTAINER Andrey Ivanov stayhardordie@gmail.com
+
+ENV REPOSITORY=https://github.com/mapsme/omim.git \
+    DIR=/srv
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    wget \
+    git \
+    libssl-dev \
+    clang \
+    libc++-dev \
+    libglu1-mesa-dev \
+    libstdc++-4.8-dev \
+    qt5-default \
```

Comment:
The Ubuntu section in the manual is about two years old Ubuntu version (14.04). Newer versions have fresh enough Qt.